docs/docs/tutorials/get-started/README.mdx

```diff
@@ -0,0 +1,101 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# ⚡ Get started
+
+## Installation
+
+<Tabs>
+
+<TabItem value="docker-compose" label="Docker Compose">
+
+Docker Compose CLI usually comes with [Docker Desktop](https://www.docker.com/products/docker-desktop).
+
+```bash
+curl -fsSL https://raw.githubusercontent.com/logto-io/logto/HEAD/docker-compose.yml | docker compose -f - up
+```
+
+After a successful composition, you will see the message like:
+
+</TabItem>
+
+<TabItem value="docker" label="Docker">
+
+1. Prepare a [PostgreSQL](https://www.postgresql.org/)@^14.0 instance。
+2. Pull the image from ghcr.io:
+
+```bash
+docker pull ghcr.io/logto-io/logto:latest
+```
+
+3. Map the container port, e.g., `3001:3001`; and set the following environment variables to the container:
+
+```yml
+ALL_YES: 1
+NO_INQUIRY: 0
+DB_URL_DEFAULT: postgres://username:password@your_postgres_url:5432 # Replace with your Postgres DSN
+```
+
+Finally run the container, you will see the message like:
+
+</TabItem>
+
+<TabItem value="script" label="Download script">
+
+**Prerequisites**
+
+- [Node.js](https://nodejs.org/) `^16.13.0`
+- [PostgreSQL](https://postgresql.org/) `^14.0`
+
+Higher versions usually work but are not guaranteed.
+
+We recommend using a new empty database which is dedicated for Logto, while it's not a hard requirement.
+
+**Download and start**
+
+In your terminal:
+
+```bash
+node -e "$(printf "%s" "$(curl -fsSL https://raw.githubusercontent.com/logto-io/logto/HEAD/install.js)")"
+```
+
+The script will download Logto and create a directory `logto` in the location you ran it. After answering [several simple questions](../../references/core/configuration.md#questions), you will see the message like:
+
+</TabItem>
```

Comment:
Do we miss a screenshot after the `see the message like:`?

Reply:
No, see the following part. They share a common “message”